Tools/autotest/models/heavy_quad.json

```diff
@@ -0,0 +1,23 @@
+# model of a large quad
+
```

Comment:
Can you please add units in the comments? Someting like:
```
 #    mass [Kg]
#     diagonal_size [m]
 # refSpd [m/s]
  # refAngle [deg]
   # refVoltage [V]
    # refCurrent  34.0 [A]
    # refAlt  [m]
    #refTempC [degC]
    # maxVoltage [V]
    # hovCurr [A]
    #refBatRes [Ohm]
    #propExpo [ ]
    #refRotRate [deg/s]
    #hoverThrOut [ ]
    #pwmMin [us]
    #pwmMax [us]
    #spin_min [ ]
    #spin_max [ ]
    #slew_max [ ]
```

Reply:
I'm going to modify the json parsing to allow embedded comments, which will make units easier

Reply:
done